### ISSUE TITLE
Sort and pagination buttons don't work in Salesforce's LWC

### ISSUE THREAD

**Issue by sjurgis** [grid-js/gridjs#247]
**Describe the bug**
Clicking on header or Next doesn't do anything. Enabling "Pause on caught exceptions" reveals:

```
TypeError: this.l[l.type] is not a function
At 

    function z(l) {
      this.l[l.type](n.event ? n.event(l) : l);
    }
```

**To Reproduce**
You'll have to setup Salesforce environment which is quite a few steps:

1. Go to [developer.salesforce.com](https://developer.salesforce.com/signup?d=70130000000td6N) to open new account. You'll get a signup link in your email. Once you create a password
2. Install [sfdx](https://developer.salesforce.com/tools/sfdxcli) which is Salesforce's CLI tool for interacting with their "orgs" aka environments.
2. Clone [this repo](https://github.com/sjurgis/gridjs-lwc) and run
```
# opens a browser where you'll have to use credentials from step 1
sfdx force:auth:web:login 
# create another env
sfdx force:org:create -s -f config/project-scratch-def.json
# push repo content onto new env
sfdx force:source:push
# disable code minification salesforce does
echo "update new User(Id = UserInfo.getUserId(), UserPreferencesUserDebugModePref=true);" | sfdx force:apex:execute
# open gridjs component in browser
sfdx force:org:open -p /lightning/cmp/c__testGridjs
```

If my steps were correct it should open page like this:
<img width="1627" alt="Screen Shot 2020-09-11 at 12 00 17 AM" src="https://user-images.githubusercontent.com/7092409/92726514-ce15e300-f3c1-11ea-9956-3edf992d16da.png">

**Comment by afshinm** [comment]
thanks @sjurgis. I'm not entirely sure if this is a Grid.js bug, but I'm happy to spend some time and debug. Is is possible to get a free trial of Salesforce's LWC (never used it before)?

**Comment by sjurgis** [comment]
Yes, step 1 will get you free trial of Salesforce, where LWC is just one of the frameworks they have. I don't think the trial ever expires.

The bug comes from a minified part of code, I assume it's going to be TS or Babel or whatever is in your build chain that adds that. Is it possible to get a build that doesn't have minified code at all?

**Comment by afshinm** [comment]
@sjurgis absolutely: https://unpkg.com/gridjs@1.17.0/dist/gridjs.development.js

**Comment by sjurgis** [comment]
@afshinm as I said, the exception traces to the part that's minified (starting ~40th line)

**Comment by afshinm** [comment]
Oh I see, I believe that's tslib minified code. thanks for clarifying, I will look into it.
